### DISCUSSION TITLE
Can ReStreamer stream to multiple sites at the same time?

### DISCUSSION THREAD

**Discussion by pvanthony** [datarhei/restreamer#519] [General]
Can ReStreamer stream to multiple sites at the same time?

Like can it stream Facebook and YouTube at the same time with one instance of ReStreamer?

**Comment by ioppermann** [comment]
@pvanthony Yes, Restreamer can stream to multiple site at the same time. We have presets for many other sites as well. Have a look at https://docs.datarhei.com/restreamer/knowledge-base/manual/publications

**Reply by pvanthony** [reply]
Thank you very much for the reply and link. I will check it out.

Good to hear that it can do streaming to multiple sites simultaneously.
